http-agent/src/main/resources/db/migration/V1_MOTECH-2502.sql

```diff
@@ -0,0 +1,3 @@
+-- change type of field "responseBody" to text
```

Comment:
Shouldn't we have two migration files? One for mysql and one for postgresql?
Is this a correct path for the migration? Aren't they placed in db/migration/mysql and db/migration/default?